learn/app-development/widgets/container/tabs.md

```diff
@@ -137,6 +137,7 @@ Page.Widgets.EmployeesTab.goToTab(2); //Navigates to sales tab
 | Tab Position | This property determines if the Tabs should be displayed on the <br> - _left_, <br> - _top (default)_, <br> - _right_ or <br> - _bottom_ of the Tab content. |
 | **Behavior** |
 | Default Pane Index | This property sets the default active pane on a load of the widget based on the provided index. The Tab Index is an integer starting from 0, 1, 2, and so on. That is, setting this property to 0 will display the first Tab pane; setting to 1 will display the second Tab pane. |
+| Retain State  | This property will allow users to maintain widget states using state handling on the URL, Local Storage or Session Storage. <br> - _URL_ : The state information will be stored in the app URL. This is the default option selected by the platform. <br> - _Local Storage_ : The widget state information will be stored in the browser local storage. <br> - _Session Storage_ : The state information will be stored in the browser session storage. <br> - _None_ : No state information will be applied to the widget.|
```

Comment:
This is enabled by default right? @SaraswathiRekhala -- We should say that.

The description should be something like - this property, that is enabled by default enables applications that are using Tabs to link specific tabs using URL rewriting approach. Other approaches to linking to specific tab includes localStorage & sessionStorgage.

Reply:
Updated as per the comments.